mutt2task.py

```diff
@@ -76,10 +93,27 @@
  
 # customize your own taskwarrior line
 # use `message' to add the subject
-res = Popen(['task', 'add', 'pri:L', '+mail', '--', message], stdout=PIPE)
-match = re.match("Created task (\d+)\.", res.stdout.read())
+if message == "None":
+    message = "E-Mail import: no subject specified."
+else:
+    message = "E-Mail subject: " + message
+
+res = Popen(['task', 'add', 'pri:L', '+email', '--', message], stdout=PIPE)
+match = re.match("^Created task (\d+).*", res.stdout.read())
 if match:
+    print(match.string.strip())
```

Comment:
Didn't you forget to remove debug message?

Reply:
Not this time. ;-)
I prefer to see what is happening, want to see the tas description of the task just added. You could make it an option to be configured, if you prefer so.

Reply:
Ok, I will check how it looks like and if decide will configure it.